playbooks/plantain_staging.yml

```diff
@@ -14,7 +14,7 @@
     - role: roles/pulibrary.nodejs
     - role: roles/pulibrary.openjdk
     - role: roles/pulibrary.blacklight-app
```

Comment:
Might this be added as a dependency to the `pulibrary.plantain` role? I would contend that its own dependencies should be run for each `plantain` server provisioning.